demos/appyx-navigation/build.gradle.kts

```diff
@@ -61,6 +61,7 @@ dependencies {
     implementation(project(":appyx-interactions:appyx-interactions"))
     implementation(project(":appyx-components:stable:spotlight:spotlight"))
     implementation(project(":appyx-components:stable:backstack:backstack"))
+    implementation(project(":appyx-components:modal:modal"))
```

Comment:
All components should go either to `stable` or to `experimental`. As a start we can add it to experimental and move it later if we feel good about it.

Reply:
Sorry, wasn't there when I created the module. Moved to experimental :)